main.js

```diff
@@ -24,15 +24,18 @@ app.on('window-all-closed', function () {
  * On ready
  */
 
-app.on('finish-launching', function () {
+app.on('ready', function () {
 	win = new BrowserWindow({
 		icon: './media/icon.png'
 	});
 
 	win.setSize(475, 400);
 	win.center();
-	win.setTitle('Imagemin ' + app.getVersion());
-	win.loadUrl('file://' + __dirname + '/index.html');
+	win.loadURL('file://' + __dirname + '/index.html');
```

Comment:
There's a better `win.loadFile` method in Electron now.

Reply:
Well, it was some time since I did these changes so I'm happy to accept any pushes to my branch =) I am currently not working with imagemin.